src/app.js

```diff
@@ -17,10 +17,19 @@ export default class App extends Component {
   constructor(props) {
     super(props);
 
+    // Only use analytics in production
```

Comment:
Analytics are useful for analyzing how much usage the app is getting. Since we can provide the `user_role` dimension to the analytics platform, we will be able to see usage by user type ("student", "faculty", "police", etc.).

During development, our usage will not be realistic (I would not actually be using the app at all if I wasn't working on it, for example), so logging that usage would throw off our overall analytics numbers. Furthermore, I often log in using a test account that has the "god" role, so if we were logging usage during development, it would look like there were very active users with the "god" role, but in reality it would just be me working on the app.